bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/SizedMetadata.hs

```diff
@@ -69,6 +69,8 @@ assumeMapCosts _proxy = stepFunction [
       ShelleyBasedEraAlonzo  -> 42
       ShelleyBasedEraBabbage -> 42
       ShelleyBasedEraConway  -> 42
+      -- TODO: check if this is correct!
+      ShelleyBasedEraDijkstra -> 42
```

Comment:
There are test cases in `bench/tx-generator/test/Main.hs` which would need to include a newly defined property (in `SizedMetadata.hs`):
```
prop_mapCostsDijkstra  = measureMapCosts AsDijkstraEra == assumeMapCosts AsDijkstraEra
```